src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped work");

```

Comment:
Similar to the `doWork` method, the message in `stopWork` should also be more informative and indicate the type of machine. For instance: 'Bulldozer has stopped working.'